galette/templates/default/pages/transactions_list.html.twig

```diff
@@ -215,7 +214,7 @@
             <td data-col-label="{{ _T("Description") }}">{{ transaction.description|escape }}</td>
             <td class="collapsing" data-col-label="{{ _T("Amount") }}">{{ transaction.amount }}</td>
             <td data-col-label="{{ _T("Payment type") }}">{{ transaction.getPaymentType() }}</td>
-            {% if mode != 'ajax' and (login.isAdmin() or login.isStaff() or login.isGroupManager()) %}
+            {% if (login.isAdmin() or login.isStaff()) and mode != 'ajax' %}
```

Comment:
Group ma,ager acces should be kept (but checked)

Reply:
Are you sure ?
Group managers cannot edit nore remove transactions.

Reply:
This access has been explicitely added in following commit: bc885c55310faf14abf7fb134ef85e514bddb91c
Goal was to allow groups managers to attach and detach contributions. I did not reference any issue; I do not remember if that was a requested feature :(